src/app/series/directives/series-templates.component.ts

```diff
@@ -12,39 +12,39 @@ import {
   template: `
     <form *ngIf="series">
     <header>
-        <h2>Audio Versions</h2>
+        <h2>Audio Templates</h2>
         <p>
-          When adding stories to your series, you can have different versions of the audio for
+          When adding episodes to your series, you can have different versions of the audio for
           distribution. For example, it is common to have a podcast version of the audio broken
           into segments for ad injection, but also a broadcast version with different breaks for
           local news, different credits, or edits and bleeps to meet FCC permitted language requirements.
-          On this page, you can define the types of audio versions that will be expected for each story in this
-          series, and basic requirements for each of those versions.
+          On this page, you can define basic templates for the versions of audio that will be expected for
+          each episode in this series, and requirements for each of those templates.
```

Comment:
minor: the comma between "series, and" seems off. I can see why there is a separation of those ideas, so maybe it would read better with the comma if it said "series, as well as". Thought it worth mentioning, but I'll defer to you on what correct grammar would be.